weave/trace_server/trace_server_interface.py

```diff
@@ -43,14 +98,20 @@ class CallSchema(BaseModel):
     output: typing.Optional[typing.Any] = None
 
     ## Summary: a summary of the call
-    summary: typing.Optional[typing.Dict[str, typing.Any]] = None
+    summary: typing.Optional[SummaryMap] = None
```

Comment:
It seems like you found the golden solution here. Worth checking with @gtarpenning and @andrewtruong to check to see if the degenerate effects they noticed are exhibited here. In Griffin's PR we decided to punt on the pydantic typing for now. IDK how much trouble vs benefit this is causing you